Recrutify/Recrutify.Services/DTOs/AppointInterviewDTO.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Recrutify.Services.DTOs
+{
+    public class AppointInterviewDTO
+    {
+        public Guid UserId { get; set; }
+
+        public Guid ProjectId { get; set; }
```

Comment:
Project id is common for all appointments, it's not different for every schedule slot. Do you appoint interviews on different projects at once?
remove it, and pass projectId as query parameter in controller and also in BulkAppointInterviewsDTO for validation, see other bulk methods for analogy